File: python/prograimg_python_basic/Exercises/conditional_statements/conditional_statements_-_exercise/world_swimming_record.py
import math

record_second = float(input())
meters = float(input())
time_second = float(input())

need_time = meters * time_second
water = math.floor(meters / 15) * 12.5

end_time = need_time + water

if end_time < record_second:
    print("Yes, he succeeded! The new world record is " + "{:.2f}".format(end_time) + " seconds.")

elif end_time >= record_second:
    need_more = end_time - record_second
    print("No, he failed! He was " + "{:.2f}".format(need_more) + " seconds slower.")

File: JS/JavaScript/JS-Basics/ConditionalStatementsExercise/WorldSwimmingRecord.js
function swimming(inputs) {
    let recordSeconds = parseFloat(inputs[0]);
    let meters = parseFloat(inputs[1]);
    let timePerMeterInSeconds = parseFloat(inputs[2]);
    
    let neededSeconds = (meters * timePerMeterInSeconds) + (Math.floor(meters / 15) * 12.5);

    if (neededSeconds < recordSeconds) {
        console.log(`Yes, he succeeded! The new world record is ${neededSeconds.toFixed(2)} seconds.`);
    }
    else {
        console.log(`No, he failed! He was ${(neededSeconds - recordSeconds).toFixed(2)} seconds slower.`);
    }
}

swimming(["55555.67",
"3017",
"5.03"])

